eox_tagging/test/test_viewset.py

```diff
@@ -75,6 +76,19 @@ def test_create_tag(self):
         response = self.client.post("/api/v1/tags/", data, format='json')
         self.assertEqual(response.status_code, 201)
 
+    def test_create_tag_without_owner(self):
+        """"Used to test creating a tag."""
```

Comment:
here there is a different condition in the input data and we want a specific response for that data, hence your docstring should show that information, with the current doc-string  my comment would be, why do we need this if the previous test does the same thing ? 
This is just convention, but separate every section with spaces https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80

Reply:
You're right.